### PR TITLE
Feat/rating form

### PR DESCRIPTION
**Pull request by Scttpr**

### CODE REVIEW

File: pages/_ride_id.vue [hunk 1]
```diff
@@ -171,6 +171,7 @@ export default {
     if (this.listener && navigator && navigator.geolocation && navigator.geolocation.clearWatch) {
       navigator.geolocation.clearWatch(this.listener);
     }
+    this.$router.push('/rating');
```
Comment (Nainterceptor) [inline pages/_ride_id.vue]: ? That not what we've in mind by "redirecting"

File: pages/rating.vue [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+          <p><em>Les champs suivis d'un * sont obligatoires</em></p>
+        </div>
+      </div>
+    </section>
+    <div class="columns is-centered">
+      <form @submit.prevent="sendForm">
+        <div class="column">
+          <ec-field
+            id="ec-form-gsbdd"
+            label="À quel GSBdD êtes-vous rattaché ? *"
```
Comment (Nainterceptor) [inline pages/rating.vue]: I think it's not longer required, if we save the original ride somewhere

File: pages/_ride_id.vue (hunk 1 above)
Comment (Scttpr) [reply pages/_ride_id.vue]: My bad, I handled redirection through sms link, added the rideId in query param from this link.

**Comment by Scttpr** [comment]
Requested changes handled

File: pages/rating.vue [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+    },
+
+    async validateForm() {
+      await this.$validator.validate();
+      return !Object.keys(this.veeFields).some((key) => this.veeFields[key].valid === false);
+    },
+
+    async sendForm() {
+      const validation = await this.validateForm();
+      if (!validation) return;
```
Comment (Rodrive) [inline pages/rating.vue]: If Eslint is okay with it, its okay I guess. Maybe we should add the curly rule

File: pages/rating.vue [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+      this.pending = true;
+
+      console.log(this.$route.query);
```
Comment (Rodrive) [inline pages/rating.vue]: Now this is something strange. Does eslint was okay with this?

Comment (Rodrive) [inline pages/rating.vue]: Now this is something strange. Does eslint was okay with this? Anyway it should be deleted

Comment (Scttpr) [reply pages/rating.vue]: It was ok but I switched back to classical curly syntax

File: pages/rating.vue (hunk 3 above)
Comment (Scttpr) [reply pages/rating.vue]: It was ok but I switched back to classical curly syntax

**Comment by Scttpr** [comment]
[Intégrer questionnaire satisfaction client au site soutenu](https://trello.com/c/U5lrwnIp/130-int%C3%A9grer-questionnaire-satisfaction-client-au-site-soutenu)

**Pull request merged**